openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -438,71 +416,59 @@ const EntityTable = ({
     );
   };
 
-  const getDataTypeDisplayCell = (record: Column | Column) => {
+  const renderDataTypeDisplay: TableCellRendered<Column, 'dataTypeDisplay'> = (
+    dataTypeDisplay
+  ) => {
     return (
       <>
-        {record.dataTypeDisplay ? (
-          <>
-            {isReadOnly ? (
-              <div className="tw-flex tw-flex-wrap tw-w-60 tw-overflow-x-auto">
-                <RichTextEditorPreviewer
-                  markdown={record.dataTypeDisplay.toLowerCase()}
-                />
-              </div>
-            ) : (
-              <>
-                {record.dataTypeDisplay.length > 25 ? (
-                  <span>
-                    <PopOver
-                      html={
-                        <div className="tw-break-words">
-                          <span>{record.dataTypeDisplay.toLowerCase()}</span>
-                        </div>
-                      }
-                      key="pop-over"
-                      position="bottom"
-                      theme="light"
-                      trigger="click">
-                      <div className="tw-cursor-pointer tw-underline tw-inline-block">
-                        <RichTextEditorPreviewer
-                          markdown={`${record.dataTypeDisplay
-                            .slice(0, 20)
-                            .toLowerCase()}...`}
-                        />
-                      </div>
-                    </PopOver>
-                  </span>
-                ) : (
-                  record.dataTypeDisplay.toLowerCase()
-                )}
-              </>
-            )}
-          </>
+        {dataTypeDisplay ? (
+          isReadOnly || (dataTypeDisplay.length < 25 && !isReadOnly) ? (
+            lowerCase(dataTypeDisplay)
+          ) : (
+            <PopOver
+              html={
+                <div className="break-word">
+                  <span>{lowerCase(dataTypeDisplay)}</span>
+                </div>
+              }
+              key="pop-over"
+              position="bottom"
+              theme="light"
+              trigger="click">
+              <Typography.Text ellipsis className="cursor-pointer">
+                {dataTypeDisplay}
+              </Typography.Text>
+            </PopOver>
+          )
         ) : (
           '--'
         )}
       </>
     );
   };
 
-  const getDescriptionCell = (index: number, record: Column | Column) => {
+  const renderDescription: TableCellRendered<Column, 'description'> = (
+    _,
+    record,
+    index
+  ) => {
```

Comment:
Can we use the `description` value directly instead of reading it from the `record`?

Reply:
Done